apps/docs/content/components/VideoPlayer.mdx

```diff
@@ -111,7 +128,7 @@ The `VideoPlayer` component exposes a `useVideo` hook that can be used to contro
 Full documentation for the `useVideo` hook can be found [below](#usevideo-context).
 
 ```tsx live
-() => {
+;() => {
```

Comment:
Looks like a typo? Could we revert this? 

<img width="457" alt="Screenshot 2025-01-08 at 12 53 18" src="https://github.com/user-attachments/assets/cff26a7d-879c-4843-bd91-d956964dfd7c" />

Reply:
Good spot, thanks. My IDE likes to put a semicolon there on save for some reason.

I've updated the example to use the `javascript live noinline` tags and refactored it slightly to prevent this happening again.